File: 1328 - Break a Palindrome.java
class Solution {
    public String breakPalindrome(String palindrome) {
        
        if (palindrome.length() <= 1) {
            return "";
        }
        
        boolean isEven = palindrome.length() % 2 == 0;
        int midpoint = palindrome.length() / 2;
        
        char[] pChar = palindrome.toCharArray();
        
        for (int i = 0; i < pChar.length; i++) {
            // Trivial case - can be an any index
            if (pChar[i] != 'a' && i != midpoint) {
                pChar[i] = 'a';
                break;
            } 
            
            if (i == midpoint) {
                pChar[palindrome.length()-1] = 'b';
                break;
            }
        }
        
        String result = String.valueOf(pChar);
        return result;
    }
}

File: 1328 - Break a Palindrome.cs
public class Solution {
    public string BreakPalindrome(string palindrome) {     
        if (palindrome.Length <= 1) {
            return "";
        }
        
        int midpoint = palindrome.Length/2;
        char[] pChar = palindrome.ToCharArray();
        
        for (int i = 0; i < pChar.Length; i++) {
            
            if (pChar[i] != 'a' && i < midpoint) {
                pChar[i] = 'a';
                break;
            }
            
            if (i == midpoint) {
                pChar[palindrome.Length-1] = 'b';
                break;
            }
        }      
        return new String(pChar);
      
    }
}